static/app/views/performance/browser/resources/utils/useResourceFilters.ts

```diff
@@ -18,7 +18,11 @@ export type ModuleFilters = {
     | 'blocking'
     | '!blocking';
   [BrowserStarfishFields.SPAN_DOMAIN]?: string;
-  [BrowserStarfishFields.SPAN_OP]?: 'resource.script' | 'resource.css' | 'resource.img';
+  [BrowserStarfishFields.SPAN_OP]?:
+    | 'resource.script'
+    | 'resource.css'
+    | 'resource.font'
+    | 'resource.img';
```

Comment:
maybe we should rename these filter values now that they dont map exactly to span op.

Maybe just remove `resource.X`?

Reply:
Yeah good point, I think we should not name them after Ops at all now. I'll work towards changes like that in upcoming clean up PRs!